framework/sessions/__init__.py

```diff
@@ -94,9 +94,16 @@ def set_previous_url(url=None):
     url = url or request.path
     if any([rule(url) for rule in settings.SESSION_HISTORY_IGNORE_RULES]):
         return
+
```

Comment:
You should be able to simply remove `set_previous_url` (including where it is used) and `goback`, searching around the code base I don't see where they provide any functionality, I'm assuming I simply missed these when we moved authentication over to CAS.